scripts/generate_hamilton_input_seq_quant_plate.py

```diff
@@ -0,0 +1,74 @@
+#!/usr/bin/env python
+import sys
+
+
+from EPPs.common import GenerateHamiltonInputEPP, InvalidStepError
+
+
+class GenerateHamiltonInputSeqQuantPlate(GenerateHamiltonInputEPP):
+    """"Generate a CSV containing the necessary information for preparing the spectramax picogreen plate. The standards locaiton is not stored in the LIMS and will be hard-coded into the
+    Hamilton method"""
+    _use_load_config = False  # prevent the loading of the config
+    # Define the column headers that will be used in the Hamilton input file
+    csv_column_headers = ['Sample ID', 'Source Plate BC', 'Source Plate Position', 'Destination Plate BC', 'Destination Plate Position']
+    # Define the output file
+    output_file_name = 'SEQ_PLATE_QUANT.csv'
+
+    # Define the number of input containers that are permitted
+    permitted_input_containers = 9
+
+    # Define the number of output containers that are permitted
+    permitted_output_containers = 1
+
+    def _generate_csv_dict(self):
+        #build a dictionary of the csv lines with the output well as the key so can be populated into the output file in the best order for straightforward import into the Hamilton method
+        #in a pattern for most efficient pipetting i.e. columns then rows
+        csv_dict = {}
+
+
+
+        # assemble a dictionary that only contains PerInput outputs for each input based on input_output_maps
+
+        input_output_maps = self.process.input_output_maps
+        input_output_perinput_dict = {}
+
+        for input_output in input_output_maps:
+            input = input_output[0]['uri']
```

Comment:
`input` is a builtin/reserved word in Python - would recommend using a different variable name

Reply:
refactored